src/index.ts

```diff
@@ -1,22 +1,28 @@
-import assert from 'assert';
-import ExtendableError from 'es6-error';
-
+import * as assert from 'assert';
+import ExtendableError from 'extendable-error';
 const isString = d => Object.prototype.toString.call(d) === '[object String]';
```

Comment:
Can you skip a line between the imports and the `isString` definition?

Reply:
Sure